locales/es/php.json

```diff
@@ -58,7 +58,7 @@
     "gte.string": "El campo :attribute debe tener como mínimo :value caracteres.",
     "if_you_delete_team_all_data_will_be_deleted": "Si usted elige borrar el equipo, todos los datos se eliminarán de manera permanente.",
     "image": "El campo :attribute debe ser una imagen.",
-    "in": "The :attribute seleccionado no es válido.",
+    "in": "El :attribute seleccionado no es válido.",
```

Comment:
Other translations begin with the words "El campo". Maybe it's needed here too.

Reply:
In the source [`'in' => 'The selected :attribute is invalid.'`](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Translation/lang/en/validation.php#L69) doesn't appear the word *field* whose meaning is *campo*.

This translation doesn't require the word "*campo*". 
It was an error in my translation so the observation as proposed is correct.

Reply:
OK